sdk/nodejs/tests/output.spec.ts

```diff
@@ -65,23 +65,25 @@ describe("output", () => {
         assert.equal(value, "inner");
     }));
 
-    it("can await even when isKnown is a rejected promise.", asyncTest(async () => {
+    it("can not await if isKnown is a rejected promise.", asyncTest(async () => {
```

Comment:
change in behavior.  however, this test was just here to catch changes, not to state that we must preserve the above behavior.

i simply wanted to ensure with the previous impl that if a rejected promise passed in we never were left with an *unresolved* promise.  i.e. in the previous impl, this was testing that our `finally` branch worked and that we would forget to set our `new Promise` explicitly in corner cases.  

Now, because we're just in a normal async function with normal promises, we don't have to worry about `new Promise` instances being leaked and never resolved/rejected.